app/controllers/organizations/organization/local-involvements/edit.js

```diff
@@ -71,8 +22,6 @@ export default class OrganizationsOrganizationLocalInvolvementsEditController ex
 
   reset() {
     this.deleteAllUnsavedLocalInvolvements();
-    this.hideTotalFinancingPercentageError();
-    this.hideMoreThanOneFinancialTypeError();
```

Comment:
**Blocker:** Need a reset call on each model.involvements in order to reset the error value